src/javascript/module-01-the-basics/11-types-booleans-and-equality/11-types-booleans-and-equality.mdx

```diff
@@ -0,0 +1,111 @@
+---
+tocTitle: 11 - Types - Booleans and Equality
+title: Types - Booleans and Equality
+slug: module-01-the-basics/types-booleans-and-equality
```

Comment:
could we take `module-` out and have the url just be `01-the-basics/...`

Reply:
`module-` has been removed from the slugs 👍